src/envs/blocks.py

```diff
@@ -102,14 +114,14 @@ def _transition_pick(self, state: State, x: float, y: float, z: float,
         next_state.set(block, "pose_y", y)
         next_state.set(block, "pose_z", self.pick_z)
         next_state.set(block, "held", 1.0)
-        next_state.set(self._robot, "fingers", fingers)
+        next_state.set(self._robot, "fingers", self.closed_fingers)
```

Comment:
do we still need the fingers feature?

Reply:
yeah we still need it for checking if the gripper is open or closed.